lib/smart_proxy_openscap/storage_fs.rb

```diff
@@ -23,7 +23,7 @@ def get_arf_html(digest)
       xml = get_arf_file(digest)[:xml]
       size = get_arf_file(digest)[:size]
       arf_object = OpenSCAP::DS::Arf.new(:content => xml, :path => 'arf.xml.bz2', :length => size)
-      arf_object.html.force_encoding('UTF-8')
+      arf_object.html
```

Comment:
Hmm! I looked at the OpenSCAP source -- I think this is a bit more complicated.

Since this is a call to a c function (via ffi wrapper), the string that comes back is probably going to be "US-ASCII" (or "ASCII-8BIT"?) This may or may not be a problem, depending on the ruby version, the encoding used in the xml document, and what sinatra and rack do with that string: ruby 1.8.7 shouldn't need any encoding manipulations -- any two strings can be concatenated, manipulated, etc, regardless of whether they are multibyte and what character codes they contain. Recent ruby 1.9.3 and later ones (any ruby vm which has default internal encoding set to 'UTF-8") may or may not have issues here (Probably not. I can't see why sinatra or rack would internally use utf-8 characters that cannot be represented in ascii), but the only way to find out is to test this with a report that contains non-ascii characters. Alternatively, you could keep String#enforce_encoding("utf-8") here, but only for non 1.8.7 rubies (I assume openscap reports always use utf-8 compatible encodings).